.github/workflows/slsa2_provenance.yml

```diff
@@ -22,7 +22,7 @@ permissions:
 #            Input and output argument definitions                #
 #                                                                 #
 ###################################################################
-on:
+"on":
```

Comment:
is this intentional?

Reply:
It's a thing that yamllint picks up on because 'on' is a keyword in yaml that resolves to a boolean "true" value. In this case I don't thinks it really a big deal since it's a key name rather than a value, but yamllint complains nontheless.

I can maybe tweak the config we use in a later PR to allow it in this case
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy

Reply:
https://github.com/slsa-framework/slsa-github-generator/issues/241